docs/source/internal/trainer_utils.rst

```diff
@@ -24,6 +24,8 @@ Utilities
 
 .. autoclass:: transformers.EvaluationStrategy
```

Comment:
It can be removed from the docs since it's deprecated: we don't want users to find it anymore and use it.